CONTRIBUTING.rst

```diff
@@ -98,14 +98,95 @@ Before you submit a pull request, check that it meets these guidelines:
 
 Tips
 ----
 
 To run a subset of tests::
 
-    $ python -m pytest tests/test_pvl.py
+	$ pytest tests/test_pvl.py
+
+
+What to expect
+--------------
+
+We want to keep development moving forward, and you should expect
+activity on your PR within a week or so.
+
+
+Rules for Merging Pull Requests
+-------------------------------
+
+Any change to resources in this repository must be through pull
+requests (PRs). This applies to all changes to documentation, code,
+binary files, etc. Even long term committers must use pull requests.
+
+In general, the submitter of a PR is responsible for making changes
+to the PR. Any changes to the PR can be suggested by others in the
+PR thread (or via PRs to the PR), but changes to the primary PR
+should be made by the PR author (unless they indicate otherwise in
+their comments). In order to merge a PR, it must satisfy these conditions:
+
+1. Have been open for 24 hours.
+2. Have one approval.
+3. If the PR has been open for 1 week without approval or comment, then it
+   may be merged without any approvals.
+
+Pull requests should sit for at least 24 hours to ensure that
+contributors in other timezones have time to review. Consideration
+should also be given to weekends and other holiday periods to ensure
+active committers all have reasonable time to become involved in
+the discussion and review process if they wish.
+
+In order to encourage involvement and review, we encourage at least
+one explicit approval from committers that are not the PR author.
+
+However, in order to keep development moving along with our low number of
+active contributors, if a PR has been open for a week without comment, then
+it could be committed without an approval.
+
+The default for each contribution is that it is accepted once no
+committer has an objection, and the above requirements are
+satisfied. 
+
+In the case of an objection being raised in a pull request by another
+committer, all involved committers should seek to arrive at a
+consensus by way of addressing concerns being expressed by discussion,
+compromise on the proposed change, or withdrawal of the proposed
+change.
+
+If a contribution is controversial and committers cannot agree about
+how to get it merged or if it should merge, then the developers
+will escalate the matter to the PlanetaryPy TC for guidance.  It
+is expected that only a small minority of issues be brought to the
+PlanetaryPy TC for resolution and that discussion and compromise
+among committers be the default resolution mechanism.
+
+Exceptions to the above are minor typo fixes or cosmetic changes
+that don't alter the meaning of a document. Those edits can be made
+via a PR and the requirement for being open 24 h is waived in this
+case.
+
+
+PVL People
+----------
+
+- A PVL **Contributor** is any individual creating or commenting
+  on an issue or pull request.  Anyone who has authored a PR that was
+  merged should be listed in the AUTHORS.rst file.  
```

Comment:
should we ask then that the PR is updating `AUTHORS.rst` accordingly already?

Reply:
Yes, there is a recommendation in the PR template for submitting authors to commit an updated AUTHORS.rst file, if they aren't already in it.